.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
+}
```

Comment:
Not sure if this is necessary to include here (its very IDE specific) and really up to the individual if they like to format on save. While I personally use it for TS, I don't use it for Rust.

Furthermore, I don't think this would be needed for the rest of the PR to work its magic.

Reply:
Agreed. I will move it to my user settings in vscode and remove this change.